language.py

```diff
@@ -526,6 +528,7 @@ def _on_lsp_msg(self, msg):
                         msg_status(f'{LOG_NAME}: {self.lang_str}: Document formatting - no info')
 
         elif msgtype == events.PublishDiagnostics:
+            msg.uri = unquote(msg.uri)
```

Comment:
@Alexey-T wait! this works for Lua diagnostics, but breaks clang diagnostics. Hm...

Reply:
fixed it